code/modules/cooking/machinery/cooking_machines/_appliance.dm

```diff
@@ -670,12 +669,11 @@
 	var/scan_rating = 0
 	var/cap_rating = 0
 
-	for(var/obj/item/stock_parts/P in component_parts)
-		if(isscanner(P))
-			scan_rating += P.rating - 1 // Default parts shouldn't change stats
-		else if(iscapacitor(P))
-			cap_rating += P.rating - 1
+	for(var/obj/item/stock_parts/capacitor/C in component_parts)
```

Comment:
Why'd you change this? The former is more consistent with the rest of the codebase, so far as I'm aware, and is better.